### DISCUSSION TITLE
Feature Request - Is it possible to allow users to customize the columns shown on the home screen?

### DISCUSSION THREAD

**Discussion by byronholloway** [openaudible/openaudible#561] [Ideas]
I would like to show the number of ratings a book has so that I and sort the most reviewed books on my list.   Also, a 5 star book rated by 10 people is less valuable feedback than a 4 star book with 10K reviews...   Also, some books are more polarizing and get lower ratings because of their content but still get a high number of reviews.    

Also, can we keep our own notes in the database?   A flag to whether it has been read or not? 

Thanks for the great product.   Best $12 I've spent in a while!!!

**Comment by openaudible** [comment]
Good idea about the table and note field. We are working on a tag editor that could include the notes field... but trying to stay focused. Feature creep is a danger. 
 
For now, export to excel and you can view and sort all the data that way.
